x-pack/plugins/apm/public/components/app/transaction_details/WaterfallWithSummmary/TransactionTabs.tsx

```diff
@@ -34,7 +34,7 @@ export function TransactionTabs({
   exceedsMax,
 }: Props) {
   const history = useHistory();
-  const tabs = [timelineTab, metadataTab, logsTab];
+  const tabs = [timelineTab, timelineNewTab, metadataTab, logsTab];
```

Comment:
The PR description mentions a new duration formatter but nothing about a new timeline component. Should this be part of the PR?

Reply:
oopss

Reply:
i pushed few commits by mistake